src/components/material/material-buttons/online/MaterialButtonOnlinePublizon.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FC } from "react";
+import { Manifestation } from "../../../../core/utils/types/entities";
+import {
+  hasPlayerManifestation,
+  hasReaderManifestation
+} from "../../../reader-player/helper";
+import MaterialSecondaryLink from "../generic/MaterialSecondaryLink";
+import MaterialSecondaryButton from "../generic/MaterialSecondaryButton";
+import { playerModalId } from "../../player-modal/helper";
+import { getManifestationIsbn } from "../../../../apps/material/helper";
+import { useModalButtonHandler } from "../../../../core/utils/modal";
+import { useText } from "../../../../core/utils/text";
+import { ButtonSize } from "../../../../core/utils/types/button";
+
+type MaterialButtonOnlinePublizonType = {
+  size?: ButtonSize;
+  manifestations: Manifestation[];
+  dataCy?: string;
+};
+
+const MaterialButtonOnlinePublizon: FC<MaterialButtonOnlinePublizonType> = ({
```

Comment:
I would maybe reconsider the naming of this component. 
Maybe `MaterialButtonsOnlineInternal` 
- adding "s" at ..Buttons.. because there's going to be more of them inside (right? the loan + try)
- ..Internal.. serving as the opposite of `MaterialButtonOnlineExternal` that are all the online materials that lead outside of our app (note that that component is in singular because it only shows a single button)

If you really don't like that naming, then I can think of something like `MaterialButtonsOnlineEReol` or `MaterialButtonsOnlineViewable`